target/scripts/build/packages.sh

```diff
@@ -0,0 +1,101 @@
+#! /bin/bash
+
+set -eE -u -o pipefail
+
+# shellcheck source=../helpers/log.sh
+source /usr/local/bin/helpers/log.sh
+
+export DEBIAN_FRONTEND=noninteractive
+QUIET='-qq' ; _log_level_is_trace && QUIET='-y'
```

Comment:
Is this the only place that `DEBIAN_FRONTEND` and `LOG_LEVEL=trace` matters?

Reply:
Yes, so we can probably remove them entirely from the Dockerfile of we use a dedicated script.